README.md

```diff
@@ -190,6 +190,26 @@ models:
 
 ```
 
+#### relationships_where ([source](macros/schema_tests/relationships_where.sql))
+This test validates the referential integrity between two tables (same as the core relationships schema test) with an added predicate to filter out some rows from the test. This is useful to exclude records such as test entities, rows created in the last X minutes/hours to account for temporary gaps due to ETL limitations, etc.
+
+Usage:
+```yaml
+version: 2
+
+models:
+  - name: model_name
+    columns:
+      - name: from_column
```

Comment:
cool test! I'd change `from_column` to `id` here, then remove the `from:` argument to the `relationships_where` test. I think that will reflect a more typical usage of a test like this.

It's really great that you included the `from` argument here as well -- that makes it possible to define a test that _does not_ apply to a particular column, eg. a test on an expression

Reply:
Good point. Done.